biomaj/workflow.py

```diff
@@ -1685,14 +1689,19 @@ def wf_uncompress(self):
                     tmpFileNameElts[len(tmpFileNameElts) - 1] = 'tmp_' + tmpFileNameElts[len(tmpFileNameElts) - 1]
                     tmpCompressedFile = self.session.get_offline_directory() + '/' + '/'.join(tmpFileNameElts)
                     archives.append({'from': origFile, 'to': tmpCompressedFile})
+                    logging.debug('Workflow:wf_tmpCompressedFile' + tmpCompressedFile)
+                    logging.debug('Workflow:wf_tmpFileNameElts:' + str(tmpFileNameElts))
                 else:
                     continue
 
                 shutil.copy(origFile, tmpCompressedFile)
 
                 not_ok = True
                 while nb_try < 3 and not_ok:
-                    status = Utils.uncompress(origFile)
+                    try:
```

Comment:
why not... but why??? uncompress should never raise an exception as it already handles itself exceptions.

so adding unnecesssary complexity in code

Reply:
uncompress is not handling exception as it's pass by .tgz without uncompressing and saying anything :(

Reply:
But uncompress should not rause an exception and only return true or false
Exception is handled at uncompress level.